distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Errors.enso

```diff
@@ -5,10 +5,23 @@ import project.Network.HTTP.HTTP_Status_Code.HTTP_Status_Code
 ## PRIVATE
    Error if the user is not logged into Enso Cloud.
 type Not_Logged_In
+    ## PRIVATE
+    Error
```

Comment:
Why is it preferred to have a constructor?

Reply:
Without the constructor the `to_display_text` method did not work correctly.

Singleton types are not specified well enough IMO in current Enso implementation and method dispatch on them is a bit wonky - I guess I could also remove the `self` argument from `to_display_text` to make it a 'static' method, but I was less sure that this solution will work right in all contexts, so it felt safer to just use the same pattern we use elsewhere - a regular atom type instead of singleton.